impeller/renderer/backend/vulkan/test/mock_vulkan.cc

```diff
@@ -410,7 +413,8 @@ VkResult vkCreateFence(VkDevice device,
                        const VkFenceCreateInfo* pCreateInfo,
                        const VkAllocationCallbacks* pAllocator,
                        VkFence* pFence) {
-  *pFence = reinterpret_cast<VkFence>(0xfe0ce);
+  MockDevice* mock_device = reinterpret_cast<MockDevice*>(device);
+  *pFence = reinterpret_cast<VkFence>(new MockFence());
```

Comment:
You need to add an override to `vkDestroyFence` too so you aren't leaking these.